packages/api-client/src/libs/create-client.ts

```diff
@@ -333,5 +338,26 @@ export const createApiClient = ({
     modalState,
     /* The workspace store */
     store,
+    /** Update the currently selected example */
+    updateExample: (exampleKey: string, operationId: string) => {
+      if (!exampleKey || !operationId) return
+
+      const request = Object.values(requests).find(
+        ({ operationId: reqOperationId }) => reqOperationId === operationId,
+      )
+      if (!request) return
+
+      const contentType =
+        Object.keys(request.requestBody?.content || {})[0] || 'application/json'
```

Comment:
This will be fine for now, but we will probably add multiple content type support soon